### ISSUE TITLE
Ask users if they want to load the autosaved data when going from View to Edit

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2266]
When users land on an `/edit` page and Form Runner find auto-saved data for that document id, it asks users whether they wanted to load the auto-saved or manually-saved data. Instead, if they land on the `/view` page and click on the *Edit* button, Form Runner shows the manually-saved data. This RFE calls for asking users which data they want to load when going from the `/view` to `/edit` page, just as we now ask them if going directly to the `/edit` page.

However, we don't want to ask users *again* in case if from `/edit` they go to `/view` and then `/edit` again. So we need to keep track of whether we asked that question already, and pass that information between `/edit` and `/view`.

**Comment by bwallis42** [comment]
Could you just explicitly delete the draft if the user answers "no" when asked if they want to continue editing it? Then the next time there wouldn't be a draft and the user wouldn't be asked the second time.

**Comment by avernet** [comment]
@bwallis42 I don't think that buys us much: even if we were to do that, we would still have to implement what is described above, as users won't be asked that question when coming in through the view page. So I prefer to only delete the auto-saved document when users manually save something.

**Comment by ebruchez** [comment]
[+1 from community again](http://discuss.orbeon.com/Autosave-Broken-in-4-9-td4660067i20.html#a4660626)

**Comment by ebruchez** [comment]
Found this in the context of of #7073. It's unclear if this is a good idea. But we now have the ability to pass more state between mode changes.

**Comment by avernet** [comment]
Re-reading this, wouldn't it make more sense to ask the user whether they want to load the autosave data when they access the view page in the first place?

This would also eliminate the complexity of tracking whether the question has already been asked when switching between view and edit modes, and therefore be more straightforward to implement.
